lib/resque_cleaner/server.rb

```diff
@@ -120,7 +120,7 @@ def text_filter(id, name, value)
           @stats = { :klass => {}, :exception => {} }
           @total = Hash.new(0)
           @jobs.each do |job|
-            klass = job["payload"]["class"] || 'UNKNOWN'
+            klass = job["payload"].to_h["class"] || 'UNKNOWN'
```

Comment:
Yeah that's what I had at first, but the issue here isn't the existence of the payload key, it's the fact that payload has a nil value.  I could change it to the following for clarity.
`
payload = job["payload"] || {}
klass = payload["class"] || 'UNKNOWN'
`